README.md

```diff
@@ -14,6 +14,31 @@ See https://github.com/openshift/enhancements/pull/565
 	- Boot the VM with that ISO
 - You can now monitor the progress using `make ssh` and `journalctl -f -u bootkube.service` or `kubectl --kubeconfig ./sno-workdir/auth/kubeconfig get clusterversion`
 
+# How to run - manual mode
+- Create a workdir for the installer - `mkdir sno-workdir`
+- Create an `install-config.yaml` in the sno-workdir. An example file can be found in `./install-config.yaml.template`
+- Download the ISO to the workdir `./download_live_iso.sh sno-workdir/base.iso`
+- Get an installer binary using `oc adm extract` or `./hack/build.sh`
```

Comment:
@eranco74 you might want to elaborate more here, feel free to edit. I wasn't sure what to write.

Reply:
`oc adm extract` isn't an option since we no longer have a release image with an up to date installer that can install single node.